alvr/client_openxr/src/interaction.rs

```diff
@@ -138,7 +138,9 @@ impl InteractionContext {
             p if p.is_quest() => QUEST_CONTROLLER_PROFILE_PATH, // todo: create new controller profile for quest pro and 3
             Platform::PicoNeo3 => PICO_NEO3_CONTROLLER_PROFILE_PATH,
             Platform::Pico4Ultra => PICO4S_CONTROLLER_PROFILE_PATH,
-            p if p.is_pico() => PICO4_CONTROLLER_PROFILE_PATH,
+            Platform::Pico4 | Platform::Pico4Pro | Platform::Pico4Enterprise => {
+                PICO4_CONTROLLER_PROFILE_PATH
+            }
```

Comment:
I think this is alright, but to give a default for pico I would use the Pico4Ultra profile. So, remove the `Platform::Pico4Ultra => PICO4S_CONTROLLER_PROFILE_PATH` line and add below here `p if p.is_pico() => PICO4S_CONTROLLER_PROFILE_PATH`